### ISSUE TITLE
Unable to install tools, try manually

### ISSUE THREAD

**Issue by kokurate** [six2dez/reconftw#887]
Hey,

I'm having an error when installing reconftw:

> Unable to install brutespray, try manually (18/46)

![image](https://github.com/six2dez/reconftw/assets/85746411/b4c9cdcb-d35e-48c0-b811-12ae29565d91)

I've tried installing it manually using Kali:
`sudo apt install brutespray`
> (Reference: https://www.kali.org/tools/brutespray/)

Is this the correct way to do it, or do we have to use Golang?

How do I manually install the tool so that reconftw will run properly, even if I install the tools manually? Could you please guide me through the steps?

**Comment by unknownghost401** [comment]
try[ ](https://github.com/x90skysn3k/brutespray)

Everything is Go Tools .. so go for it and install it

**Comment by six2dez** [comment]
Not everything but most of them are, the most simple way I would say is searching for the tool name on Google :)

**Comment by kokurate** [comment]
> Not everything but most of them are, the most simple way I would say is searching for the tool name on Google :)

Of course, I searched first as a rule before asking, but I just want to make sure that installing the package inside Kali will not prevent reconftw from running properly. Anyway, thank you for creating these tools, really appreciate it!
